### PR TITLE
Add xcuitest

### PR DESCRIPTION
**Pull request by SelimGribaa**
Asana Task : https://app.asana.com/0/1141396442496489/1198995533528106

### CODE REVIEW

File: Sandbox/SandboxUITests/LoginWithEmailSuccess.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
+//
+//  SandboxUITests.swift
+//  SandboxUITests
+//
+//  Created by admin on 17/12/2020.
+//  Copyright © 2020 Reachfive. All rights reserved.
+//
```
Comment (thomasreach5) [inline Sandbox/SandboxUITests/LoginWithEmailSuccess.swift:7]: Can we remove those?

File: Sandbox/SandboxUITests/LoginWithEmailSuccess.swift [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+import XCTest
+
+class LoginWithEmailSuccess: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
```
Comment (thomasreach5) [inline Sandbox/SandboxUITests/LoginWithEmailSuccess.swift:14]: Can we remove these comments?

File: Sandbox/SandboxUITests/LoginWithEmailSuccess.swift [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        loginButton.tap()
+        
+        sleep(2)
+        let updateEmail = app.buttons["email"]
+        XCTAssert(updateEmail.exists)
+
+        
+        // Use recording to get started writing UI tests.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    }
```
Comment (thomasreach5) [inline Sandbox/SandboxUITests/LoginWithEmailSuccess.swift:51]: Do we have to go through the Sandbox app to test library functions ? I think we could implement those tests so they could run without starting the app.

File: Sandbox/SandboxUITests/LoginWithFaceBookSuccess.swift [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testLoginWithFaceBook() throws {
+        // Use recording to get started writing UI tests.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+        let app = XCUIApplication()
```
Comment (thomasreach5) [inline Sandbox/SandboxUITests/LoginWithFaceBookSuccess.swift:31]: If I understand correctly the app is started for each test ?

**Review by thomasreach5 — COMMENTED** [review]
I'm pretty confident we could implement those tests without resorting to going through the app.

File: IdentitySdkCore/IdentitySdkCore/Classes/ReachFivePassword.swift [hunk 5]
```diff
@@ -2,12 +2,12 @@ import Foundation
 import BrightFutures
 
 public extension ReachFive {
-    func signup(profile: ProfileSignupRequest, scope: [String]? = nil) -> Future<AuthToken, ReachFiveError> {
+    func signup(profile: ProfileSignupRequest,redirectUrl: String? = nil, scope: [String]? = nil) -> Future<AuthToken, ReachFiveError> {
         let signupRequest = SignupRequest(
             clientId: sdkConfig.clientId,
             data: profile,
             scope: (scope ?? self.scope).joined(separator: " "),
-            acceptTos: nil
```
Comment (thomasreach5) [inline IdentitySdkCore/IdentitySdkCore/Classes/ReachFivePassword.swift:10]: This should have been done in a separate PR, it's completely unrelated to tests.

Comment (SelimGribaa) [reply IdentitySdkCore/IdentitySdkCore/Classes/ReachFivePassword.swift]: This PR was based on an old PR (Add the redirectURL parameter in the signup method) (https://github.com/ReachFive/identity-ios-sdk/pull/74) which was merged
